smartsim/settings/base.py

```diff
@@ -158,6 +163,27 @@ def add_exe_args(self, args):
                 raise TypeError("Executable arguments should be a list of str")
             self.exe_args.append(arg)
 
+    def set(self, arg, value=None):
+        """allows users to set individual run arguments
```

Comment:
let's expand on this docstring with a couple examples in code blocks.

Reply:
Expanded in https://github.com/CrayLabs/SmartSim/pull/166/commits/a2d3a9414cced0fcc3a32b89229b54abf3fcbfdf with examples in code blocks